rotkehlchen/history/events.py

```diff
@@ -536,3 +538,24 @@ def fail_history_cb(error_msg: str) -> None:
             defi_events,
             ledger_actions,
         )
+
+    def query_kraken_staking_events(
+            self,
+            filter_query: HistoryEventFilterQuery,
+            only_cache: bool,
+    ) -> List[StakingEvent]:
+        """Queries staking events for kraken.
+        If only_cache is given then only trades cached in the DB are returned.
+        No service is queried.
+        """
+        exchanges_list = self.exchange_manager.connected_exchanges.get(Location.KRAKEN)
+        if exchanges_list is None:
+            return []
+
+        if only_cache is not True:
+            for kraken_instance in exchanges_list:
+                kraken_instance.query_kraken_ledgers(start_ts=Timestamp(0), end_ts=ts_now())  # type: ignore  # noqa: E501
```

Comment:
no. Extract start_ts and end_ts from the filter query like we do in other calls.

Reply:
Didn't know this existed. Thanks!